site_root/input/resources/examples/PractitionerRole-example-practitioner-id.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
This resource seems to be a duplicate of the below (site_root/input/resources/examples/PractitionerRole-example-practitioner-role-id.json)? If so, please remove it.

Reply:
resolved.